src/sentry/notifications/helpers.py

```diff
@@ -150,11 +177,13 @@ def get_values_by_provider_by_type(
     ],
     all_providers: Iterable[ExternalProviders],
     type: NotificationSettingTypes,
+    organization: "Organization",
 ) -> Mapping[ExternalProviders, NotificationSettingOptionValues]:
     """
     Given a mapping of scopes to a mapping of default and specific notification
     settings by provider, determine the notification setting by provider for
     the given notification type.
+    TODO(mgaeta): Remove `organization` parameter.
```

Comment:
did you mean to remove it in this PR?

Reply:
No we only need the organization for the feature flag so when the experiment is done we should take out the parameter.